events/pony.txt

```diff
@@ -118,12 +118,13 @@ province_event = {
 		change_religion = selenite
 		add_province_modifier = {
 			name = "religious_zeal_at_conv"
-			duration = 7000
+			duration = 5000
```

Comment:
The spread is really aggressive. I know adjusting the months from 200 to 500 does a lot to it, but the zeal is a pain, and this might help it along as well. After all, in the equestrian world, the two godly entities are much more solid concepts than just different interpretations of the usual monotheistic god of today. I imagine less fanatic beliefs, when the god actually can get into your face going "you what mate?". Perhaps even lower it even further for that reason, but now I'm rambling